jupyter-extension/src/plugins/mount/mount.tsx

```diff
@@ -138,24 +138,18 @@ export class MountPlugin implements IMountPlugin {
       'pfs',
     );
 
-    this._exploreScreen = new SplitPanel({orientation: 'vertical'});
```

Comment:
I think the SplitPanel was what made the the PFS browser viewable. You might want to consider going back to if you can't get the Panel to work